src/C1T2X_OBU.py

```diff
@@ -137,7 +140,22 @@ def VANET_listening_thread():
 			c1t2x_logger.debug("Received %s from VANET", pkt)
 			if pkt:
 				if not parseVANETPacket:
-					sendLAN(pkt[0])
+					# Check if ack or payload
+					data = pkt[0].decode('utf-8')
+					if data == "1":  # Ack
+						with mutex:
+							wait_for_ack = False
+						c1t2x_logger.info("Received ack")
+					elif data == previous_packet_received:  # Duplicate message received, so just resend ack
+						ack = b"1"
```

Comment:
Moved the ack definition out of the try block. I think "1" should be fine for the ack, since we need a simple message that indicates a message was successfully received. I'm open to other ideas but this seems the most intuitive to me.